Client/Frontend/Fakespot/FakespotOptInCardViewModel.swift

```diff
@@ -132,6 +135,20 @@ struct FakespotOptInCardViewModel {
         return finalString
     }
 
+    private func processStringBasedOnLocale(_ inputString: String) -> String {
+        let localizedDot = String(inputString.last!)
+        switch Locale.current.languageCode {
+        case "de", "fr":
+            return inputString.components(separatedBy: localizedDot)
```

Comment:
My understanding is this code removes the last sentence from the first paragraph. If this is wanted and not separate strings, looks good from me too.

Reply:
This sounds like a potential for trouble: We do not know if that the translations have the same amount of paragraphs so we could be removing too little. I think it would be better to have two separate translations for this instead of manipulating the existing translation.